twitter_scraper/README.md

```diff
@@ -0,0 +1,18 @@
+# smag-twitterscraper
+
+- [about](#about)
+- [getting started](#getting-started)
+
+## about
+Reads user_names from kafka topic `user_names`, scrapes and inserts user-profiles (kafka-topic: `users_scraped`) and tweets (kafka-topic: `tweets_scraped`).  
+Then inserters (not there yet) read these information from kafka and insert them into a postgres database.
```

Comment:
`(not there yet)` now they exist, right?

Reply:
@Urhengulas

Reply:
solved by https://github.com/codeuniversity/smag-mvp/pull/16/commits/84161d84016ec17ba9a3a71df76055ffae0d37cb